functions/package-lock.json

```diff
@@ -6,6 +6,7 @@
     "": {
```

Comment:
If there isn't a reason for this `package-lock.json` to be pushed I think it should be deleted